checks.d/docker_daemon.py

```diff
@@ -249,6 +254,21 @@ def check(self, instance):
         if self.collect_container_size:
             self._report_container_size(containers_by_id)
 
+        if self.collect_container_count:
+            self._report_container_count_by_state(containers_by_id)
+
+        if self.collect_dead_container_count:
+            self._report_container_count_by_state(containers_by_id, state="Dead")
+
+        if self.collect_exited_container_count:
+            self._report_container_count_by_state(containers_by_id, state="Exited")
+
+        if self.collect_volume_count:
+            self._report_volume_count()
+
+        if self.collect_dangling_volume_count:
+            self._report_volume_count(filters={'dangling': True})
```

Comment:
AFAICT, the resulting JSON we receive back does not include information about whether the volume is dangling. We cannot therefore filter based on the response of the first function call.